doc/release/release_0.20.rst

```diff
@@ -0,0 +1,479 @@
+Announcement: scikit-image 0.20.0
+=================================
+
+We're happy to announce the release of scikit-image v0.20.0!
+
+scikit-image is an image processing toolbox for SciPy that includes algorithms
+for segmentation, geometric transformations, color space manipulation,
+analysis, filtering, morphology, feature detection, and more.
+
+For more information, examples, and documentation, please visit our website:
+
+https://scikit-image.org
+
+
+New features and improvements
+-----------------------------
+- Several footprint generating and consuming functions in ``skimage.morphology`` now support footprint decomposition.
+  By decomposing a footprint into several smaller ones, morphological operations can potentially be speed up.
+  The decomposed footprint can be generated with the new ``decomposition`` parameter of the functions ``rectangle``, ``diamond``, ``disk``, ``cube``, ``octahedron``, ``ball``, ``octagon`` in ``skimage.morphology``.
+  The ``footprint`` parameter of the functions ``binary_erosion``, ``binary_dilation``, ``binary_opening``, ``binary_closing``, ``erosion``, ``dilation``, ``opening``, ``closing``, ``white_tophat``, and ``black_tophat`` in ``skimage.morphology`` now accepts a tuple of a footprint and the number of times it should be applied iteratively.
+  (`#5482 <https://github.com/scikit-image/scikit-image/pull/5482>`_, `#6151 <https://github.com/scikit-image/scikit-image/pull/6151>`_)
+- Added support for the Modified Hausdorff Distance (MHD) metric in ``skimage.metrics.hausdorff_distance`` which performs better than the directed Hausdorff Distance (HD).
+  (`#5581 <https://github.com/scikit-image/scikit-image/pull/5581>`_)
+- Added support for the Modified Hausdorff Distance (MHD) metric in ``skimage.metrics.hausdorff_distance`` via the new parameter ``method`` .
+  (`#5581 <https://github.com/scikit-image/scikit-image/pull/5581>`_)
+- Added two datasets ``skimage.data.protein_transport`` and ``skimage.data.nickel_solidification``. (`#6087 <https://github.com/scikit-image/scikit-image/pull/6087>`_)
+
+Changes and new deprecations
+----------------------------
+
+Completed deprecations
+----------------------
+
+Bugfixes
+--------
+
+Documentation
+-------------
+- Add a textbook-like tutorial on measuring fluorescence at the nuclear envelope.
+  (`#5262 <https://github.com/scikit-image/scikit-image/pull/5262>`_)
+
+Other and development related updates
+-------------------------------------
+- Added benchmarks for ``morphology.local_maxima``. (`#3255 <https://github.com/scikit-image/scikit-image/pull/3255>`_)
+- Fix the autogeneration of API docs for lazy loaded subpackages. (`#6075 <https://github.com/scikit-image/scikit-image/pull/6075>`_)
+- Checkout gh-pages with a shallow clone. (`#6085 <https://github.com/scikit-image/scikit-image/pull/6085>`_)
+
+TODO
+----
+- Transplant the change of scikit-learn into scikit-image for RANSAC  (`#6046 <https://github.com/scikit-image/scikit-image/pull/6046>`_)
```

Comment:
```suggestion
- Incorporate RANSAC improvements from scikit-learn into scikit-image (`#6046 <https://github.com/scikit-image/scikit-image/pull/6046>`_)
```

Reply:
Thanks. I haven't gotten to these Todo items yet and will be gradually moving them over to their correct section. :+1: